examples/postgres/advanced-blog-cli/Cargo.toml

```diff
@@ -5,11 +5,11 @@ edition = "2018"
 publish = false
 
 [dependencies]
-bcrypt = "0.10.1"
+bcrypt = "0.14.0"
```

Comment:
Could we switch that to [argon2](https://lib.rs/crates/argon2) as that seems to be the state of art password hashing function?

Reply:
sure